imdb/__init__.py

```diff
@@ -124,6 +143,75 @@ def get_published_date(info):
     return date.fromisoformat(info.datePublished)
 
 
+def save_image(movie_id, url, horizontal=False):
+    """
+    Save image to cloud storage
+
+    :param str movie_id: title id of the imdb
+    :param bool horizontal: horizontal or not
+    :return: upload status
+    :rtype: bool
+    """
+    if not url:
+        return False
+
+    temp_path = f'{config.system.temporary}/images'
+
+    # create temporary path if not exists
+    if not os.path.exists(temp_path):
+        os.makedirs(temp_path)
+
+    name = f'{movie_id}_horizontal' if horizontal else f'{movie_id}'
+
+    jpg_path = f'{temp_path}/{name}.jpg'
+    webp_path = f'{temp_path}/{name}.webp'
+
+    # get image content by using image url
+    headers = {
+        'Accept-Encoding': 'identity',
+        'Connection': 'Keep-Alive',
+    }
+
+    bs = requests.get(url, stream=True, headers=headers)
+
+    with open(jpg_path, 'wb') as f:
+        # write image content to local file
+        for b in bs:
+            f.write(b)
+
+    img = Image.open(jpg_path).convert('RGB')
+    width, height = img.size
+
+    # resize image
+    if width >= MAX_WIDTH and height >= MAX_HEIGHT:
```

Comment:
I believe, this one should be "or", not?

Reply:
Yes, you're right sir.